omnibus/files/private-chef-cookbooks/Berksfile

```diff
@@ -1,4 +0,0 @@
-source 'https://supermarket.chef.io'
-
-cookbook 'enterprise', git: 'https://github.com/chef-cookbooks/enterprise-chef-common', tag: 'v1.1.0'
```

Comment:
enterprise is a dep in the main cookbook and there's no reason to get it from git other than wasting time in the build process so remove it from here.